dbms/src/Common/CPUAffinityManager.h

```diff
@@ -115,8 +117,9 @@ class CPUAffinityManager
 #endif
 
     // unused except Linux
-    [[maybe_unused]] int query_cpu_percent;
-    [[maybe_unused]] int cpu_cores;
+    UNUSED_MEMBER int query_cpu_percent;
```

Comment:
Is it really unused? We'd better add symbol like `maybe`.

Reply:
addressed